lib/seattleflu/id3c/cli/command/etl/clinical.py

```diff
@@ -215,7 +215,9 @@ def encounter_details(document: dict) -> dict:
                 "FluShot": flu_shot(document.get("FluShot")),
                 "AssignedSex": [sex(document.get("AssignedSex"))],
                 "HispanicLatino": hispanic_latino(document.get("HispanicLatino")),
-                "MedicalInsurance": insurance(document.get("MedicalInsurance"))
+                "MedicalInsurance": insurance(document.get("MedicalInsurance")),
+                "AdmitDuringThisEncounter": document.get("admit_during_this_encounter"),
+                "AdmitToICU": document.get("admit_to_icu"),
```

Comment:
For consistency, these values should probably be converted to "yes/no" using a function (similar to `FluShot`).